install-dev/data/db_structure.sql

```diff
@@ -1325,7 +1325,7 @@ CREATE TABLE `PREFIX_order_detail` (
   `product_id` int(10) unsigned NOT NULL,
   `product_attribute_id` int(10) unsigned DEFAULT NULL,
   `id_customization` int(10) unsigned DEFAULT 0,
-  `product_name` varchar(255) NOT NULL,
+  `product_name` text NOT NULL,
```

Comment:
It seems that the field `name` in `ps_product_lang` is `varchar(128)`, can this one be longer?

Reply:
@atomiix `ps_product_lang` has a base product name, `product_name` in this table is a combination of the base product name + variant groups and values :)

Reply:
@atomiix we can extend the length of the base product name if needed, but I don't recall such a need, this PR is related to the automatically generated name that is used for the historical data (in case of deleting the product, etc.)